### PR TITLE
Start asyncing the localization code

### PR DESCRIPTION
**Pull request by mcoughlin**

### CODE REVIEW

File: skyportal/handlers/api/gcn.py [hunk 1]
```diff
@@ -209,6 +218,24 @@ def delete(self, dateobs):
...
+            .filter(
+                Localization.id == localization_id,
+            )
+            .first()
+        )
+        localization = get_contour(localization)
+        DBSession().add(localization)
+        DBSession().commit()
+    except Exception as e:
+        return request_handler.error(f"Unable to generate contour: {e}")
```
Comment (stefanv) [inline skyportal/handlers/api/gcn.py:234]: We cannoy send  responses back to the originating request, which has completed.

Comment (mcoughlin) [reply skyportal/handlers/api/gcn.py]: Ah yes. Now emulating you.

File: skyportal/handlers/api/gcn.py [hunk 2]
```diff
@@ -209,6 +221,24 @@ def delete(self, dateobs):
...
+            .filter(
+                Localization.id == localization_id,
+            )
+            .first()
+        )
+        localization = get_contour(localization)
+        DBSession().add(localization)
+        DBSession().commit()
+    except Exception as e:
+        return log(f"Unable to generate contour: {e}")
```
Comment (stefanv) [inline skyportal/handlers/api/gcn.py:237]: I would also print the localization ID here so that we can inspect it.

**Review by stefanv — APPROVED** [review]
Excellent!  One more change and I think we're good to go.

File: skyportal/handlers/api/gcn.py (hunk 2 above)
Comment (mcoughlin) [reply skyportal/handlers/api/gcn.py]: Yes great idea. added

File: skyportal/handlers/api/gcn.py [hunk 3]
```diff
@@ -209,6 +221,26 @@ def delete(self, dateobs):
...
+        )
+        localization = get_contour(localization)
+        DBSession().add(localization)
+        DBSession().commit()
+    except Exception as e:
+        return log(
+            f"Unable to generate contour for localization {localization_id}: {e}"
+        )
+    finally:
+        DBSession.remove()
```
Comment (dmitryduev) [inline skyportal/handlers/api/gcn.py:241]: Nice!

File: skyportal/handlers/api/gcn.py [hunk 4]
```diff
@@ -209,6 +221,26 @@ def delete(self, dateobs):
...
+                Localization.id == localization_id,
+            )
+            .first()
+        )
+        localization = get_contour(localization)
+        DBSession().add(localization)
+        DBSession().commit()
+    except Exception as e:
+        return log(
+            f"Unable to generate contour for localization {localization_id}: {e}"
```
Comment (dmitryduev) [inline skyportal/handlers/api/gcn.py:238]: Log username/id?

**Review by dmitryduev — APPROVED** [review]
LGTM!

**Comment by dmitryduev** [comment]
CI failure apparently unrelated.

**Pull request merged**